TraceScripts/README.md

```diff
@@ -60,4 +60,36 @@ The numeric values of the "logAreaFlags_ApiInOut", "logAreaFlags_FnInOut", and "
 2. Enable **FileSharing** for your application, this can be done through the **Info.plist** file of your application.
 3. Copy the `PlayFabMultiplayerLogger.json` file into the `~/Documents` directory.
 4. Run the application.
-5. Locate the output log inside the directory specified by the "logFolder" property of `PlayFabMultiplayerLogger.json`.
\ No newline at end of file
+5. Locate the output log inside the directory specified by the "logFolder" property of `PlayFabMultiplayerLogger.json`.
+
+### Android: Instructions
+
+> **Note: Gathering PlayFabMultiplayer Logs on Android only works for Android API 29 and below, due to Google's app-scoped storage permission changes for Android API 30 and beyond. This will be fixed in a future release.**
+
+> *Note: Relevant documentation for copying files to and from Android devices can be found [here](https://developer.android.com/tools/adb#copyfiles).*
+
+1. Download the [`PlayFabMultiplayerLogger.json` file](Android/PlayFabMultiplayerLogger.json) from this repo.
+2. Connect the Android device to host machine running Android Studio/Android Debug Bridge (adb).
+3. Enable USB Debugging on your Android device ([relevant documentation](https://developer.android.com/studio/debug/dev-options#Enable-debugging)).
+4. Open a new terminal that has ADB. Double-check that your device is readable from ADB with 
+```
+$> adb devices -l
+```
+5. Create your `config` directory on the Android device.
+
+    a. Please note that you need to create this directory before pushing the config file to your Android device due to ADB limitations.
+
+    b. You'll also need to create the directories one level at a time due to ADB limitations.
```

Comment:
```suggestion
    a. Please note that you need to create this directory before pushing the config file to your Android device due to ADB limitations.
    b. You'll also need to create the directories one level at a time due to ADB limitations.
```

unnecessary newline between the (a) and (b) subbullets?

Reply:
It looks that way in the markdown itself, but I need to add a second newline in order for it to show properly in the markdown renderer.

Reply:
I think what that suggests is that maybe markdown doesn't recognize a/b to be proper subbullets

consider switching to just using regular bullets or numbered bullets again

using VS Code's markdown renderer as one data point it seems to render things incorrectly only when using a/b

![image](https://github.com/PlayFab/PlayFabMultiplayer/assets/43357130/d133c81e-254d-4ede-99d3-5e606ecbd3eb)

Reply:
this is marked resolved but I don't see the changes in the latest push

Reply:
oh wait now it is :P